### ISSUE TITLE
[BUG]: On Linux, shortcut creation is failing because template is missing

### ISSUE THREAD

**Issue by Guts** [Guts/qgis-deployment-cli#320]
### Is there an existing issue or a documentation page for this?

- [X] I have searched the existing issues and within documentation

### Which version of the app are you using?

0.26.0

### Operating System

Linux Ubuntu*

### Packaging mode

Stand-alone executable (.i.e qdt.exe)

### Description of the bug

QDT uses [a template file](https://github.com/Guts/qgis-deployment-cli/blob/61b92d10a0d2ca2168cff3ea3ec8317640a9cd47/qgis_deployment_toolbelt/profiles/shortcut_freedesktop.template) to make Desktop and Start menu shortcuts.

This file seems to be missing in the packaged executable.

### Steps to reproduce

1. Download executable for Linux from latest release
1. make it runnable:

  ```sh
  chmod +x QGISDeploymentToolbelt_0-26-0_Ubuntu22-04_64bit_Python3-11-4
  ```

1. run it with scenario demo: 

  ```sh
  ./QGISDeploymentToolbelt_0-26-0_Ubuntu22-04_64bit_Python3-11-4 -s https://github.com/Guts/qgis-deployment-cli/raw/main/examples/scenarios/demo-scenario.qdt.yml -vvv
  ```

### Relevant log output

```Shell
[...]
2023-11-08 12:32:38||DEBUG||job_shortcuts||218||Icon set in profile.json exists so it will be used: /home/jmo/.local/share/QGIS/QGIS3/profiles/qdt_demo/images/logo_qdt.ico
2023-11-08 12:32:38||INFO||check_path||106||/tmp/_MEIoNV7N4/qgis_deployment_toolbelt/profiles/shortcut_freedesktop.template doesn't exist. Attempt 1/2. Try again with user and vars expansion.
2023-11-08 12:32:38||ERROR||bouncer||40||/tmp/_MEIoNV7N4/qgis_deployment_toolbelt/profiles/shortcut_freedesktop.template doesn't exist. Attempt 2/2. Game over.
Traceback (most recent call last):
  File "qgis_deployment_toolbelt/commands/deployment.py", line 201, in run
  File "qgis_deployment_toolbelt/jobs/job_shortcuts.py", line 153, in run
  File "qgis_deployment_toolbelt/profiles/shortcuts.py", line 183, in create
  File "qgis_deployment_toolbelt/profiles/shortcuts.py", line 347, in freedesktop_create
  File "qgis_deployment_toolbelt/utils/check_path.py", line 235, in check_path
  File "qgis_deployment_toolbelt/utils/check_path.py", line 107, in check_path_exists
  File "qgis_deployment_toolbelt/utils/check_path.py", line 114, in check_path_exists
FileExistsError: /tmp/_MEIoNV7N4/qgis_deployment_toolbelt/profiles/shortcut_freedesktop.template doesn't exist. Attempt 2/2. Game over.
/tmp/_MEIoNV7N4/qgis_deployment_toolbelt/profiles/shortcut_freedesktop.template doesn't exist. Attempt 2/2. Game over.
```

**Comment by Guts** [comment]
@sigeal is one of the bugs you encountered?

**Comment by Guts** [comment]
Fix shipped with version 0.27.0: https://github.com/Guts/qgis-deployment-cli/releases/tag/0.27.0
